File: Python3/프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    end = len(num_list) - 1
    end1 = end - 1
    
    if num_list[end] > num_list[end1]:
        num_list.append(num_list[end] - num_list[end1])
    else:
        num_list.append(num_list[end] * 2)
    
        
    return num_list

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.java
class Solution {
    public int[] solution(int[] num_list) {
        int tmp = num_list.length - 1; // 3 -> 2
        int answer[] = new int[tmp + 2]; // 3 -> 4
        for (int i = 0; i < num_list.length; i++) {
            answer[i] = num_list[i];
        }
        
        
        if (num_list[tmp] > num_list[tmp-1]) {
            answer[tmp + 1] = num_list[tmp] - num_list[tmp-1];
        }
        else answer[tmp + 1] = num_list[tmp] * 2;
        
        return answer;
    }
}